build_config.rb

```diff
@@ -8,7 +8,7 @@ def gem_config(conf)
   conf.gem mgem: 'mruby-io',             checksum_hash: '86d58a2d3727f556e6e6ce10d170b6170911a263'
   conf.gem mgem: 'mruby-open3',          checksum_hash: '211550775492e68ba217526808cf1df9deb7fa5a'
   conf.gem mgem: 'mruby-shellwords',     checksum_hash: '2a284d99b2121615e43d6accdb0e4cde1868a0d8'
-  conf.gem mgem: 'mruby-specinfra',      checksum_hash: '4132571dec892ba4962f98766ee891e8466e3a47'
+  conf.gem mgem: 'mruby-specinfra',      checksum_hash: ''
```

Comment:
TODO: set checksum_hash of mruby-specinfra after this p-r k0kubun/mruby-specinfra#3 is merged.